### ISSUE TITLE
Interpolation does not work in a plastic tool.

### ISSUE THREAD

**Issue by jrhyme** [opentoonz/opentoonz#3318]
**Describe the bug**
During animation, interpolation behaves inappropriately

**To Reproduce**
List steps to reproduce the behavior.
1. In the settings, linear interpolation
2. Constant interpolation works in the first two keys
3. Changing to another, nothing happens.

**Expected behavior**
I want the interpolation to work the way I tell her

**Screenshots**
![bandicam-2020-05-30-21-22-32-962](https://user-images.githubusercontent.com/49364531/83337109-7bf0fc00-a2c1-11ea-9b93-b4f734e663cd.gif)

**Device Information:**
 - OS: Windows 10
 - OpenToonz Version: 1.4 [e.g. Nightly (2020-29-05)]
 - GPU: [e.g. rx 580]

**Comment by beeheemooth** [comment]
Indeed, the interpolation for the bone parameter keys of the Plastic Tool cannot be changed from the context menu or in the Function Editor (meaning that you want to change the interpolation of several parameters of all bones). The interpolation of animations made using the Animate Tool can be changed from the context menu, but it is also impossible to change several selected parameters in the Function Editor. Only the parameter that you selected first changes. Most likely this is not a glitch, but an imperfection.
![ezgif-4-932f0e7bee3e](https://user-images.githubusercontent.com/25724133/83339670-fda05400-a2d8-11ea-9766-c1258f72c997.gif)
